### PR TITLE
Consume leaderboard api with async and await

### PR DESCRIPTION
**Pull request by SimbaWANGU**
In this Pull Request, the following list of requirements were achieved: 
1. Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
2. Created a new game with the name of your choice by using the API.
3. Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
4. Implemented the form "Submit" button (sending data to the API).
5. Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/refresh.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const refresh = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/CDJ8oUTB9lzH5vIwdnUz/scores');
+  const results = await response.json();
+  return results.result;
```
Comment (Cathella) [inline src/modules/refresh.js:4]: ✅ Used async and await with the fetch method to send and receive data 👍🏻

File: src/index.js [hunk 2]
```diff
@@ -15,11 +15,38 @@ import {
...
 
 const component = () => {
   mainContainer.appendChild(title);
 
   headingDiv.appendChild(subTitle);
+
+  refreshButton.addEventListener('click', () => {
+    if (unorderedList.innerHTML === '') {
+      refresh()
+        .then((response) => {
```
Comment (Cathella) [inline src/index.js:29]: ✅ Usage of **arrow functions** instead of the `function` keyword 👍🏻

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @SimbaWANGU 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Professional README.
✅ Descriptive pull request.
✅ App runs as expected. You can submit user data and it reflects on the scoreboard.


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimbaWANGU** [comment]
Thank you @Cathella for the review!
Happy coding 😄

**Pull request merged**